src/analysis/ConfigurationExt.hpp

```diff
@@ -46,6 +49,11 @@ namespace espressopp {
 
       int nextId();
       const RealND nextProperties();
+      /* increment particle iterator, get pid, get properties (pos,v)   */
+      void nextParticle();
+      size_t Id();
+      const RealND Properties();
+
       //Real3D nextCoordinates();
       //Real3D nextVelocities();
```

Comment:
Yes, it looks like that, again just kept as it was this part.
I guess previousely was started a nextCoordinates(), nextVelocities(), etc. and then opted for a single method Properties() and nextProperties().
Again, I can remove it and push again.